apps/state/test/state/stops_on_route_test.exs

```diff
@@ -363,6 +429,18 @@ defmodule State.StopsOnRouteTest do
       # know that there are only three items.
       assert [_, _, _] = merge_ids(stop_ids)
     end
+
+    test "can use an override to order individual stops" do
+      stop_ids =
+        Enum.shuffle([
```

Comment:
This will make the tests only fail intermittently if there's a problem.
Perhaps you should always do a merge in both directions, and check that both work or just do the one order that'd be broken now.